Content.Client/Chat/UI/ChatBox.xaml.cs

```diff
@@ -467,8 +477,15 @@ private void InputKeyBindDown(GUIBoundKeyEventArgs args)
 
         private void InputOnTextChanged(LineEdit.LineEditEventArgs obj)
         {
+            if(_timing.RealTime.Subtract(TimeSinceType) >= TimeSpan.FromSeconds(5))
+            {              
+                _entityManager.EventBus.RaiseEvent(EventSource.Local, new ClientChatInputActiveMessage());
+                TimeSinceType = _timing.RealTime;
```

Comment:
TimeSinceType should probably be a field on the client TypingIndicatorSystem, and instead of raising an empty event here the chatbox should just resolve the client system directly and call a method to reset TimeSinceType

Reply:
that or store the timesincetype on a networked component